feature/bgp/bgp_isis_redistribution/otg_tests/bgp_isis_redistribution_test/bgp_isis_redistribution_test.go

```diff
@@ -459,16 +472,11 @@ func verifyNonMatchingCommunityTelemetry(t *testing.T, dut *ondatra.DUTDevice, a
 	if commSet == nil {
 		t.Errorf("Community set is nil, want non-nil")
 	}
-	if deviations.BgpCommunityMemberIsAString(dut) {
-		cm := nonMatchingCommunityVal
```

Comment:
hi,
why are we removing this deviation; can we not be add the relevant changes which exercises that is not part of deviation?